llvm/tools/llvm-exegesis/lib/X86/Target.cpp

```diff
@@ -939,7 +939,8 @@ std::vector<MCInst> ExegesisX86Target::setRegTo(const MCSubtargetInfo &STI,
       }
       break;
     case 64:
-      if (STI.getFeatureBits()[X86::FeatureBWI]) {
+      if (STI.getFeatureBits()[X86::FeatureBWI] &&
+          STI.getFeatureBits()[X86::FeatureEVEX512]) {
```

Comment:
Unfortunately there is no implementation in case of `FeatureBWI && !FeatureEVEX512`, so it falls into `return {}; // Not yet implemented`. I haven't found any tests checking such case. So, decided not to add a test.

Reply:
Ah. Didn't realize that. Thanks for looking.